### ISSUE TITLE
【bug】chinese is not support in cmdDelimiter  ['{', '}']

### ISSUE THREAD

**Issue by bulolo** [guigrpa/docx-templates#107]
![image](https://user-images.githubusercontent.com/10026498/79133511-bb7d8a80-7dde-11ea-8f53-5cb465f58241.png)

![image](https://user-images.githubusercontent.com/10026498/79133197-33978080-7dde-11ea-9a93-c3eae2c59896.png)

```
 const report = await createReport({
        template: res.data,
        data:{
          '姓名':'hong',
          '标题':'junyao'
        },
        cmdDelimiter: ['{', '}']
      });
```
chinese is not support in cmdDelimiter ['{', '}']

when i  change to english , it's ok .

but the default way below is support chinese.

![image](https://user-images.githubusercontent.com/10026498/79134005-92112e80-7ddf-11ea-834e-ace7dd14c144.png)

**Comment by jjhbw** [comment]
Interesting. I'll investigate.

**Comment by jjhbw** [comment]
See referenced commit.

I have investigated a bit, and it seems that the delimiters aren't the problem.

The problem is that if you dont specify an explicit insert command (`INS` or `=`), the library will check if the command is alphanumeric using a regex. Chinese characters are not alphanumeric, so they won't qualify.

Currently, this means:

```
# Works
{=姓名}
{=标题}

# Works
{INS 姓名}
{INS 标题}

# Does not work
{姓名}
{标题}
```

I have proposed a tentative solution in #118 , but need to test this a bit more thoroughly to ensure no other behaviour is affected.

**Comment by jjhbw** [comment]
Fix is on master (620cebb385341851ea2de33de2baaec3896a1a4f) and will be part of the next release. Thanks again for your report!
